rest-api/model/database.py

```diff
@@ -38,7 +37,6 @@ def __init__(self, url, **kwargs):
     # connections after this period. (See DA-237.) To change the db wait_timeout (seconds), run:
     # gcloud --project <proj> sql instances patch rdrmaindb --database-flags wait_timeout=28800
     self._engine = create_engine(url, pool_recycle=3600, **kwargs)
```

Comment:
Don't you need to add pool_pre_ping=True?

Reply:
_Monday strikes again_

Yes. Yes you do.